src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+import createGameBoards from "./boardView.js";
+import {UltimateBoard} from "./Boards/UltimateBoard.js";
+import {HumenPlayer} from "./Players/HumenPlayer.js";
+import {ComputerPlayer} from "./Players/ComputerPlayer.js";
+import { Board } from "./Boards/Board.js";
+
+const   boardSize = 3,
+        elementsID = {gameStatus: 'gameStatus',
+                    humPlayersign: 'humPlayerSign',
+                    compPlayersign: 'compPlayerSign'};
+let humPlayer,
```

Comment:
Again, this file should be for a class, the variables you have here act as state which proves the need for OOP approach